src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService.Rewriter.cs

```diff
@@ -34,6 +34,24 @@ public override SyntaxNode Visit(SyntaxNode node)
                 return base.Visit(node);
             }
 
+            public override SyntaxNode VisitMemberAccessExpression(MemberAccessExpressionSyntax node)
+            {
+                // Ignore name node in member access expression, since we can't replace it with the new variable.
+                // For example:
+                //
+                //  class C
+                //  {
+                //      C c;
+                //      void Test()
+                //      {
+                //          var x = [|c|].c.c;
+                //      }
+                //  }
```

Comment:
i feel like the comment needs more explanation. i.e. that even though the first 'c' binds to the property, and we're introducing a variable for it, we don't want *other* references to that property *though* that local to be updated as well.  

Also, i think the example is too complicated.  `c.c` would be enough to show the problem right?  I think the extra `.c` isn't necessary (and makes it more confusing).

Reply:
Fixed